docs/release-notes/hpc-launcher-resource-pools.rst

```diff
@@ -0,0 +1,9 @@
+:orphan
+
+**New Feature**
+
+*  HPC: Add support for launcher-provided resource pools.
```

Comment:
Traditionally, this has been "Cluster:"  for things in master.yaml.

And it probably should reference Determined enterprise edition like our previous release notes.

https://hpe-mlde.determined.ai/latest/release-notes.html?highlight=release%20notes

like: 
`Cluster: The enterprise edition of [HPE Machine Learning Development Environment](https://www.hpe.com/us/en/solutions/artificial-intelligence/machine-learning-development-environment.html) can now be deployed on a PBS cluster. When using PBS scheduler, HPE Machine Learning Development Environment delegates all job scheduling and prioritization to the PBS workload manager. This integration enables existing PBS workloads and HPE Machine Learning Development Environment workloads to coexist and access all of the advanced capabilities of the PBS workload manager. You can use either Singularity or Podman for the container runtime.`

Reply:
Done.